### ISSUE TITLE
Errors running on Azure VM

### ISSUE THREAD

**Issue by rocketstack-matt** [SaveTheChildrenUK/CostOfTheDiet#54]
- Create a Win 10 Pro VM in Azure Portal (Standard D2as v5 (2 vcpus, 8 GiB memory))
- Login as admin user
- Install as admin user 

![fail-on-first-launch](https://github.com/SaveTheChildrenUK/CostOfTheDiet/assets/66839492/77cf348a-13b5-4dee-8896-ac9b0074eb86)

- db3 file exists 

![db3-file](https://github.com/SaveTheChildrenUK/CostOfTheDiet/assets/66839492/a4702aaa-0011-41f9-ba1d-bef4c9f910cd)

- DevTools Errors

```
Uncaught ReferenceError: jQuery is not defined
    at jquery.dataTables.min.js:18
    at jquery.dataTables.min.js:18
main-es2015.b536b69….js:1 
Yv
handleError	@	main-es2015.b536b69….js:1
main-es2015.b536b69….js:1 
Yv
error:
error: SyntaxError: Unexpected token < in JSON at position 0 at JSON.parse (<anonymous>) at XMLHttpRequest.a (http://localhost:8001/main-es2015.b536b6904f5a8a2a3609.js:1:2670985) at l.invokeTask (http://localhost:8001/polyfills-es2015.ea5fefa274a3df845a11.js:1:7199) at Object.onInvokeTask (http://localhost:8001/main-es2015.b536b6904f5a8a2a3609.js:1:2422327) at l.invokeTask (http://localhost:8001/polyfills-es2015.ea5fefa274a3df845a11.js:1:7120) at i.runTask (http://localhost:8001/polyfills-es2015.ea5fefa274a3df845a11.js:1:2608) at u.invokeTask [as invoke] (http://localhost:8001/polyfills-es2015.ea5fefa274a3df845a11.js:1:8249) at h (http://localhost:8001/polyfills-es2015.ea5fefa274a3df845a11.js:1:20210) at XMLHttpRequest.p (http://localhost:8001/polyfills-es2015.ea5fefa274a3df845a11.js:1:20534)
text: "<!DOCTYPE html>\r\n<html lang=\"en\">\r\n<head>\r\n\r\n  <link rel=\"stylesheet\" type=\"text/css\" href=\"/assets/css/material.indigo-pink.min.css\">\r\n  <link rel=\"stylesheet\" type=\"text/css\" href=\"/assets/css/all.css\">\r\n  <link rel=\"stylesheet\" type=\"text/css\" href=\"/assets/css/jquery.dataTables.min.css\"/>\r\n  <link rel=\"stylesheet\" type=\"text/css\" href=\"/assets/css/common.css\"/>\r\n  <script type=\"text/javascript\" src=\"/assets/js/bootstrap.bundle.min.js\"></script>\r\n  <script type=\"text/javascript\" src=\"/assets/js/jquery-3.5.1.min.js\"></script>\r\n  <script type=\"text/javascript\" src=\"/assets/js/jquery.dataTables.min.js\"></script>\r\n  <script type=\"text/javascript\" src=\"/assets/js/popper.min.js\"></script>\r\n  <script type=\"text/javascript\" src=\"/assets/js/material.min.js\"></script>\r\n\r\n  <meta charset=\"utf-8\"/>\r\n  <title>Cost of The Diet</title>\r\n  <base href=\"/\"/>\r\n\r\n  <meta name=\"viewport\" content=\"width=device-width, initial-scale=1\"/>\r\n  <link rel=\"icon\" type=\"image/x-icon\" href=\"favicon.ico\"/>\r\n<link rel=\"stylesheet\" href=\"styles.47d8d2dfd1508e8bc2ba.css\"></head>\r\n<body>\r\n  <app-root>Loading...</app-root>\r\n<script src=\"runtime-es2015.d7d2011503ee9d572f4e.js\" type=\"module\"></script><script src=\"runtime-es5.d7d2011503ee9d572f4e.js\" nomodule defer></script><script src=\"polyfills-es5.0648e8903b914ad0e867.js\" nomodule defer></script><script src=\"polyfills-es2015.ea5fefa274a3df845a11.js\" type=\"module\"></script><script src=\"main-es2015.b536b6904f5a8a2a3609.js\" type=\"module\"></script><script src=\"main-es5.b536b6904f5a8a2a3609.js\" nomodule defer></script></body>\r\n</html>\r\n"
__proto__:
constructor: ƒ Object()
hasOwnProperty: ƒ hasOwnProperty()
isPrototypeOf: ƒ isPrototypeOf()
propertyIsEnumerable: ƒ propertyIsEnumerable()
toLocaleString: ƒ toLocaleString()
toString: ƒ ()
valueOf: ƒ valueOf()
__defineGetter__: ƒ __defineGetter__()
__defineSetter__: ƒ __defineSetter__()
__lookupGetter__: ƒ __lookupGetter__()
__lookupSetter__: ƒ __lookupSetter__()
get __proto__: ƒ __proto__()
set __proto__: ƒ __proto__()
headers: Fv
lazyInit: ()=> {…}
lazyUpdate: null
normalizedNames: Map(0) {}
__proto__: Object
message: "Http failure during parsing for http://localhost:8001/api/assessments/"
name: "HttpErrorResponse"
ok: false
status: 200
statusText: "OK"
url: "http://localhost:8001/api/assessments/"
__proto__: Hv
constructor: class Yv
__proto__: Object
```

- Uninstalled and re-installed for all users. Same result.

**Comment by DavidKavanagh** [comment]
This issue appears to be related to the formatting of dates. We will fix this issue shortly, but until then setting your region to United Kingdom should provide a workaround:
![image](https://github.com/SaveTheChildrenUK/CostOfTheDiet/assets/20168917/5912ee4d-86c2-4019-b813-6170083da668)

**Comment by DavidKavanagh** [comment]
This should be resolved in release v3.0.1
